test_jit_fuser.py

```diff
@@ -0,0 +1,893 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import functools
+import os
+import unittest
+import sys
+import torch
+import torch.autograd.function as function
+from torch import Tensor
+
+from common_utils import TestCase, run_tests, IS_WINDOWS, TEST_WITH_UBSAN, \
+    skipIfRocm, skipIfNoLapack, suppress_warnings, load_tests, IS_SANDCASTLE, \
+    freeze_rng_state, set_rng_seed, slowTest
+from common_nn import module_tests, new_module_tests, criterion_tests
```

Comment:
A lot of these imports are not actually used in this file. Good to remove them. ;) 

I think this PR doesn't conflict with my change ( hopefully :P ) since my PR was mainly touching nn_functional_tests and autograd_method_tests which are not used here.  Cheers!

Reply:
thank!